build.gradle

```diff
@@ -6,19 +6,26 @@ buildscript {
     }
     dependencies {
         classpath "com.android.tools.build:gradle:4.1.2"
+        implementation 'com.github.euphony-io:euphony:562a8fab8d'
```

Comment:
I think it should be written like this.
```
    implementation 'euphony.lib:euphony:0.7.1.6'
```
Because euphony-speaker is doing that.